Testing System/src/main/java/com/epam/practice/testingsystem/data/dto/Deadline.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.practice.testingsystem.data.dto;
+
+import lombok.Data;
+import lombok.NonNull;
+
+import java.time.LocalDate;
+
+@Data
+public class Deadline {
+    @NonNull
+    private Test test;
+
+    @NonNull
+    private UniversityGroup universityGroup;
+
+    private LocalDate deadline;
+
+    public Deadline() {
+    }
+
+    public Deadline(@NonNull Test test, @NonNull UniversityGroup universityGroup, @NonNull LocalDate deadline) {
```

Comment:
Consider using `@AllArgsConstructor` and `@NoArgsConstructor` annotations instead.

Reply:
Yes, it seems more reasonable